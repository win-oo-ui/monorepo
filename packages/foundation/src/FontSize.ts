interface FontSizeType {
  xxxs: "xxxs";
  xxs: "xxs";
  xs: "xs";
  sm: "sm";
  base: "base";
  lg: "lg";
  xl: "xl";
  xxl: "xxl";
  xxxl: "xxxl";
}

const fontSizes: FontSizeType = {
  xxxs: "xxxs",
  xxs: "xxs",
  xs: "xs",
  sm: "sm",
  base: "base",
  lg: "lg",
  xl: "xl",
  xxl: "xxl",
  xxxl: "xxxl",
};

export default Object.freeze(fontSizes);




