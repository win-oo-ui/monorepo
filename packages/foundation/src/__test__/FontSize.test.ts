import FontSize from "../FontSize";

test("Snapshop of fontsizes: Foundation", () => {
  expect(FontSize).toMatchSnapshot();
});
