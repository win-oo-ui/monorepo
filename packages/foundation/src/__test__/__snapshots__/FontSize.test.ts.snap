// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Snapshop of fontsizes: Foundation 1`] = `
{
  "base": "base",
  "lg": "lg",
  "sm": "sm",
  "xl": "xl",
  "xs": "xs",
  "xxl": "xxl",
  "xxs": "xxs",
  "xxxl": "xxxl",
  "xxxs": "xxxs",
}
`;
